src/Tizen.NUI/src/public/BaseComponents/DrawableView.cs

```diff
@@ -0,0 +1,215 @@
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// DrawableView allows drawing with OpenGL. You can render to a Window directly.
+    /// DrawableView creates a context.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class DrawableView : View
+    {
+        /// <summary>
+        /// The parameter of the RenderFrame Callback.
+        /// It has data to render directly.
+        /// </summary>
+        public struct RenderCallbackInput
+        {
```

Comment:
could you please check the above warnings?
if you think this message can be ignored,
you can suppress them by using
```c#
 [global::System.Diagnostics.CodeAnalysis.SuppressMessage(ReasonType, Descriptions)]
```